default-config.neon

```diff
@@ -8,6 +8,8 @@ includes:
 parameters:
     level: 5
 
+    tmpDir: ../../../cache/addons/rextstan/phpstan
```

Comment:
Die Änderung funktioniert so nur, solange kein custom path provider verwendet wird. Muss ich drüber nachdenken

Reply:
Ich habe leider nicht rausgefunden warum ab PHP 8.x in phpstan fix `c:\xampp\tmp` verwendet wird.
Muss irgendwo in phpstan bzw. einem verwendeten vendor passieren.

Reply:
ich denke dass in der php.ini irgendwo als temp pfad dieser pfad eingetragen ist